lib/models/blueprint.js

```diff
@@ -1031,8 +1031,8 @@ let Blueprint = CoreObject.extend({
     let packageNamesAndVersions = packages.map(pkg => {
```

Comment:
This map seems out of place to me. Perhaps it should be a forEach that pushes to both `packageNames` and `packageNamesAndVersions` arrays.